nautobot/core/tests/test_views_utils.py

```diff
@@ -98,3 +100,43 @@ def test_check_filter_for_display(self):
         # with self.assertEqual(
         #     check_filter_for_display(device_filter_set_filters, "manufacturer", ["fake_slug"]), expected_output
         # )
+
+
+class CheckCountRelatedSubquery(TestCase):
+    def test_count_related(self):
+        """Assert that InventoryItems with the same Manufacuturers do not cause issues in count_related subquery."""
+        location = Location.objects.filter(parent__isnull=False).first()
+        manufacturer = Manufacturer.objects.create(name="New Manufacturer")
+        devicetype = DeviceType.objects.create(manufacturer=manufacturer, model="Device Type 1")
+        devicerole = Role.objects.get_for_model(Device).first()
+        device_status = Status.objects.get_for_model(Device).first()
+        device1 = Device.objects.create(
+            device_type=devicetype,
+            role=devicerole,
+            name="TestDevice1",
+            status=device_status,
+            location=location,
+        )
+        self.manufacturer_1 = Manufacturer.objects.create(name="Manufacturer 1")
+        self.manufacturer_2 = Manufacturer.objects.create(name="Manufacturer 2")
+        self.manufacturer_3 = Manufacturer.objects.create(name="Manufacturer 3")
```

Comment:
Probably don't need to create manufacturers? Aren't they being created by factories?